public/api/v1/testOverview.php

```diff
@@ -228,8 +243,9 @@
     if ($total_runs === 0) {
         continue;
     }
-    // Only include tests that failed at least once.
-    if ($test['failed'] === 0 && $test['timeout'] === 0) {
+    // Unless the user specified otherwise, only include tests that failed
```

Comment:
This comment can be a little clearer.  Maybe?

    Only include tests that failed at least once unless the user requests all tests.